### PR TITLE
Fix bugs in partner report cleanup script, increase debug output

### PR DESCRIPTION
**Pull request by bmedx**

### CODE REVIEW

File: tubular/google_api.py [hunk 1]
```diff
@@ -349,7 +352,10 @@ def walk_files(self, top_folder_id, file_fields='id, name', mimetype=None, recur
 
         while folders_to_visit:
             current_folder = folders_to_visit.pop()
+            print("Current folder: {}".format(current_folder))
             visited_folders.append(current_folder)
+            extra_kwargs = {}
```
Comment (bmedx) [inline tubular/google_api.py:357]: This was the first bug, this wasn't being reset so the paging parameter was being sent when it wasn't expected.

File: tubular/google_api.py [hunk 2]
```diff
@@ -349,7 +352,10 @@ def walk_files(self, top_folder_id, file_fields='id, name', mimetype=None, recur
 
         while folders_to_visit:
             current_folder = folders_to_visit.pop()
+            print("Current folder: {}".format(current_folder))
```
Comment (bmedx) [inline tubular/google_api.py]: Oops. I'll fix this to a `LOG.info`

File: tubular/scripts/delete_expired_partner_gdpr_reports.py [hunk 3]
```diff
@@ -102,10 +102,11 @@ def delete_expired_reports(config_file, google_secrets_file, age_in_days):
     try:
         delete_before_dt = datetime.now(UTC) - timedelta(days=age_in_days)
         drive = DriveApi(config['google_secrets_file'])
+        LOG('DriveApi configured')
         drive.delete_files_older_than(
             config['drive_partners_folder'],
             delete_before_dt,
-            mimetype='application/csv',
+            mimetype='text/csv',
```
Comment (bmedx) [inline tubular/scripts/delete_expired_partner_gdpr_reports.py:109]: This was the second bug, no files were being found with `application/csv` this change found what we're looking for.

**Review by doctoryes — APPROVED** [review]
:shipit:

**Pull request merged**